src/lib.rs

```diff
@@ -865,10 +1085,18 @@ impl<'a> RenderPass<'a> {
         wgn::wgpu_render_pass_set_blend_color(self.id, &color);
     }
 
+    /// Sets the active index buffer.
+    ///
+    /// Subsequent calls to `draw_indexed()` on this `RenderPass` will use `buffer` as the source
+    /// index buffer.
     pub fn set_index_buffer(&mut self, buffer: &Buffer, offset: BufferAddress) {
         wgn::wgpu_render_pass_set_index_buffer(self.id, buffer.id, offset);
     }
 
+    /// Sets the active vertex buffers.
+    ///
+    /// Each element of `buffer_pairs` describes a vertex buffer and an offset in bytes into that
+    /// buffer.
```

Comment:
we should be using `BufferAddress` here. Also, we need to document that the offsets have to be aligned to 4. (Actually, it might require bigger alignment, but I don't have the exact here. Saying 4 is safe for now.

Reply:
Should I make the `BufferAddress` change in a separate PR?

Reply:
sure thing

Reply:
see https://github.com/gfx-rs/wgpu-rs/pull/11/commits/96e8784098076e6491cb8179e6fd01f51a62574f#diff-b4aea3e418ccdb71239b96952d9cddb6R1098